### ISSUE TITLE
YChanEx Errors\Fchan.download.log doesn't create itself

### ISSUE THREAD

**Issue by SalamanderSandwich** [murrty/YChanEx#2]
YChanEx doesn't check to see if the folder or log exists before trying to write to it. I should probably open another issue for the error its giving me but I'm just getting a vague NullReferenceException. 
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at YChanEx.fourChan.download()
```
What should I do to get more details?

**Comment by murrty** [comment]
To get more details (since I'm shit at reporting informative errors), it's easier just to download the source and debug it yourself, following the steps that produced the error.

However, I do know of one issue it may be referencing; When downloading Thread.html (or some pictures from the thread), it'll throw an error (that I honestly can't remember at the moment). It happens even when the file has been created & disposed from the program (Check: [Classes/Controller.cs@L238](https://github.com/murrty/YChanEx/blob/master/Classes/Controller.cs#L238)). I'm not sure the reasoning, it just... happens. It might be stepping over itself.

**Comment by murrty** [comment]
Fixed in v1.69
